_posts/2020-12-01-introducing-the-library.md

```diff
@@ -3,7 +3,8 @@ layout: post
 title:  "Introducing the Library by StreamSounds"
 author: ogjunkyard
 categories: [ about us ]
-image: "assets/images/logo.png"
+image-location: "assets/images/"
```

Comment:
Move to settings.

Reply:
Since this has been moved to _config.yml, do we still need this setting in the post?